docs/pages/application-access/getting-started.mdx

```diff
@@ -117,43 +161,23 @@ $ sudo teleport configure \
   --app-uri=http://localhost:3000
 ```
   
-Make sure to update `--app-name` and `--app-uri` accordingly if you're using your
-own web application.
-
-The `--token` flag points to the file on the Application Service host where we
-stored the token that we generated earlier.
+- Set the `--proxy` command-line option to your Teleport Proxy Service address 
+(for example, `teleport.example.com`) or Teleport Cloud tenant 
+(for example, `mytenant.teleport.sh`).
+- Set the `--token` command-line option to the file location on the Application 
+Service host where you saved the invitation token that you generated earlier.
+- Change the `--app-name` and `--app-uri` command-line options if you're configuring 
+access to a different web application.
 
 (!docs/pages/includes/start-teleport.mdx service="the Teleport Application Service"!)
-
 </TabItem>
 <TabItem label="Kubernetes cluster">
 
-## Step 2/4. Start Grafana
-
-We've picked Grafana for this tutorial since it's very easy to run with zero
-configuration required. If you have another web application you'd like to
-expose, skip over to **Step 3**. Install Grafana with these helm instructions
-and it will be available at `http://example-grafana.example-grafana.svc.cluster.local`
-within the Kubernetes cluster.
-
-```code
-$ helm repo add grafana https://grafana.github.io/helm-charts
-$ helm repo update
-$ helm install example-grafana grafana/grafana \
-    --create-namespace \
-    --namespace example-grafana
-```
-
-## Step 3/4. Install and configure Teleport
-
-  (!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
+(!docs/pages/kubernetes-access/helm/includes/helm-repo-add.mdx!)
 
 <ScopedBlock scope={["oss", "enterprise"]}>
```

Comment:
What's the difference between this `ScopedBlock` and the one below it? Should we merge them into one?